### PR TITLE
New version: joncampbell123.DOSBox-X version 2023.10.06

### PR DESCRIPTION
**Pull request by vedantmgoyal9**
###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/125180)

### CODE REVIEW

**Comment by HunterZ** [comment]
Would like to see someone work this, since the current latest version available in winget is also broken such that it is always gets listed as an available update (see #127278).

**Review by HunterZ — APPROVED** [review]

**Comment by HunterZ** [comment]
😒 Bump

File: manifests/j/joncampbell123/DOSBox-X/2023.10.06/joncampbell123.DOSBox-X.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  InstallerUrl: https://github.com/joncampbell123/dosbox-x/releases/download/dosbox-x-v2023.10.06/dosbox-x-windows-2023.10.06-setup.exe
+  InstallerSha256: 43ACE131E3FDBD2BF69B06E20767B38CA9B8D6B2E8245365516B98FA6C503916
+  InstallerSwitches:
+    Custom: /ALLUSERS
+- Architecture: x86
+  Scope: machine
+  InstallerUrl: https://github.com/joncampbell123/dosbox-x/releases/download/dosbox-x-v2023.10.06/dosbox-x-windows-2023.10.06-setup.exe
+  InstallerSha256: 43ACE131E3FDBD2BF69B06E20767B38CA9B8D6B2E8245365516B98FA6C503916
+  InstallerSwitches:
+    Custom: /ALLUSERS
```
Comment (fancybody) [inline manifests/j/joncampbell123/DOSBox-X/2023.10.06/joncampbell123.DOSBox-X.installer.yaml:38]: @vedantmgoyal2009
```suggestion
Installers:
- Architecture: x86
  Scope: user
  InstallerUrl: https://github.com/joncampbell123/dosbox-x/releases/download/dosbox-x-v2023.10.06/dosbox-x-windows-2023.10.06-setup.exe
  InstallerSha256: 43ACE131E3FDBD2BF69B06E20767B38CA9B8D6B2E8245365516B98FA6C503916
  InstallerSwitches:
    Custom: /CURRENTUSER
- Architecture: x86
  Scope: machine
  InstallerUrl: https://github.com/joncampbell123/dosbox-x/releases/download/dosbox-x-v2023.10.06/dosbox-x-windows-2023.10.06-setup.exe
  InstallerSha256: 43ACE131E3FDBD2BF69B06E20767B38CA9B8D6B2E8245365516B98FA6C503916
  InstallerSwitches:
    Custom: /ALLUSERS
- Architecture: x64
  Scope: user
  InstallerUrl: https://github.com/joncampbell123/dosbox-x/releases/download/dosbox-x-v2023.10.06/dosbox-x-windows-2023.10.06-setup.exe
  InstallerSha256: 43ACE131E3FDBD2BF69B06E20767B38CA9B8D6B2E8245365516B98FA6C503916
  InstallerSwitches:
    Custom: /CURRENTUSER
- Architecture: x64
  Scope: machine
  InstallerUrl: https://github.com/joncampbell123/dosbox-x/releases/download/dosbox-x-v2023.10.06/dosbox-x-windows-2023.10.06-setup.exe
  InstallerSha256: 43ACE131E3FDBD2BF69B06E20767B38CA9B8D6B2E8245365516B98FA6C503916
  InstallerSwitches:
    Custom: /ALLUSERS

```

**Review by fancybody — CHANGES_REQUESTED** [review]

**Comment by HunterZ** [comment]
@vedantmgoyal2009 please review so that I stop getting notifications from user-hostile bots threatening to close the issue due to inactivity.

**Comment by HunterZ** [comment]
@vedantmgoyal2009 please review so that I stop getting notifications from user-hostile bots threatening to close the issue due to inactivity.

**Comment by HunterZ** [comment]
@ImJoakim @stephengillie can we get some attention on this please? The user-hostile bots keep trying to close it because it's waiting for reviewer attention.

**Review by ImJoakim — APPROVED** [review]

**Comment by HunterZ** [comment]
@fancybody this successfully results in the 2023.10.06 version being installed, but then Winget still insists that 2023.09.01 is installed on the next update check for some reason?

**Comment by stephengillie** [comment]
This is because the package is temporally versioned, but is in semantic format. Were the dots swapped for another delineator, these should be interpreted as strings instead, allowing temporal versioning to work correctly.

**Pull request merged**